metadata.json

```diff
@@ -68,6 +72,10 @@
     {
       "name": "puppetlabs/yumrepo_core",
       "version_requirement": ">=1.0.3 < 2.0.0"
+    },
+    {
+      "name": "puppetlabs/powershell",
+      "version_requirement": ">=2.3.0 <3.0.0"
```

Comment:
Technically `puppet-datadog-agent` supports some older puppets like `4.6.2`, I believe powershell `>=2.3.0` only officially supports `>=4.7.x`. That said, I don't think this is a problem, it's a new feature, and those older puppets are EOL. It should be fine, if anything breaks with older puppets on windows, to affirm we do *NOT* support `<4.7.0` on win?